README.md

```diff
@@ -29,12 +32,13 @@ For documentation about the Jellyfin app, see [here](https://github.com/jellyfin
 
 3. Run the command below, replacing first argument with the IP of your Samsung TV
 
+   - If you just want to install the default build, do not put anything after the IP address.
    - (Optional) You can provide preferred [jellyfin-tizen-builds](https://github.com/jeppevinkel/jellyfin-tizen-builds) option (Jellyfin / Jellyfin-TrueHD / Jellyfin-master / Jellyfin-master-TrueHD / Jellyfin-secondary) as second argument. By default, Jellyfin option is used.
    - (Optional) You can provide preferred [jellyfin-tizen-builds releases](https://github.com/jeppevinkel/jellyfin-tizen-builds/releases) release tag URL as third argument. By default, latest version is used. This is useful if you want to install older Jellyfin Tizen Client version.
-   - If you do not want to use either of these options and just install the default build, do not put anything after the IP address.
+   
 
 ```bash
-docker run --rm ghcr.io/georift/install-jellyfin-tizen <samsung tv ip> <build option> <tag url>
+**docker run --rm ghcr.io/georift/install-jellyfin-tizen** <u>samsung tv ip</u> [build option] [tag url]
```

Comment:
Could you revert the `<u>` and `**` changes please? I don't think they render correctly in github. But I might be mistaken.

I like the `[]` denoting option though! 🙂